### DISCUSSION TITLE
Add an option to auto show UI on connected

### DISCUSSION THREAD

**Discussion by cjcr** [webspiderteam/Bluetooth-DMM-For-Windows#43] [Ideas]
Hello

I will be great to have an option to show UI automatically when a device is connected in background (when the app is in taskbar or taskbar icon). So you don't need to touch the computer to see the meter UI.

Regards.

**Comment by webspiderteam** [comment]
Actually this is already done but I thought maybe with auto connection setting it can ignore. I will check it now

**Comment by webspiderteam** [comment]
Done ;) https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/releases/tag/WPFRelease1.25

**Reply by cjcr** [reply]
Now working! perfect. Thank you.

**Reply by webspiderteam** [reply]
Never mind 😊

**Reply by cjcr** [reply]
What about adding a feature to hide window to taskbar when connection to bluetooth is lost IF before was opened when bluetooth connected?
